stories/index.js

```diff
@@ -74,6 +75,14 @@ storiesOf('List Components/SingleList', module)
     template: '<base-single-list :subProps="{ sortBy: \'asc\' }"/>',
   }));
 
+
+storiesOf('Range Components/MultiRange', module)
+  .add('Basic', () => ({
+    components: { BaseMultiRange },
+    template: '<base-multi-range :subProps="{ showFilter: false }"/>',
+  }))
+
```

Comment:
@divyanshu-rawat Can you add more stories here

Reply:
@bietkul will do it.

Reply:
@bietkul updated PR with more stories.

Reply:
Let me send new PR netlify shows some err.